platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml

```diff
@@ -145,12 +151,19 @@ network:
           grpc:
             port: 9092
                 
-        orderer:
-          name: orderer
-          type: orderer          
+        orderers:
```

Comment:
Add a comment that "# This sample has multiple orderers as an example. You can use a single orderer for most production implementations."

Reply:
This is not addressed